private/numerics/functions/elliptic.rkt

```diff
@@ -0,0 +1,237 @@
+#lang racket/base
+
+;; Consult "Numerical Computation of Real or Complex Elliptic
+;; Integrals" Carlson (1994) http://arxiv.org/pdf/math/9409227v1.pdf
+;;
+;; The algorithm in scmutils seems similar to what is in that paper,
+;; but slightly different.
+;;
+;; Consider also exporting under the more familiar names Rf, Rj, Rc,
+;; Rd. Is there any input method that gets those names as subscripts.
+
+(require
+ (only-in mechanics π π/2 machine-ε))
+
+(require
+ (only-in racket/math cosh tanh))
+
+(require
+ (only-in racket/fixnum fx< fx+))
+
+(define (Rf x y z)
+  (define ε (expt (machine-ε) (/ 1 6)))
```

Comment:
Yea I have to re-add all of the ascii stuff. Can you think of a good way to automate checking that we export all the same names that they do? They have those `compile.scm` files, but I can't think of anything better than searching `scmutils` for the corresponding compile file? and checking we have a `provide` for every one of their `define`s. That said tho, I can't tell how they use namespacing. Their top level `load.scm` is probably the thing to look at for this.